test/docker-compose.yaml

```diff
@@ -21,7 +21,7 @@ services:
       - .env
     restart: always
     networks:
-      idunion_local:
+      local:
```

Comment:
I would like a more specific name for the network, e.g. indy-nodes-local or something like that

Reply:
good point :+1: